### ISSUE TITLE
epicgames captcha not working

### ISSUE THREAD

**Issue by Teufel0406** [vogler/free-games-claimer#283]
captcha not working its say 
incorrect response redresh the page

**Comment by Teufel0406** [comment]
ps refresh did not help

**Comment by rezytijo** [comment]
I got it that too, best way i choice is linked you EGS to Steam Account and login using steam scan QR Code

**Comment by biast12** [comment]
i have been having this issue too, but running the tool as headless (not run it via docker) seems to work, it doesn't even prompt a captcha

**Comment by fl1ckzeu** [comment]
are u using windows/linux?

**Comment by biast12** [comment]
windows, but should be the same as long as you you node and the right node_modules

**Comment by Teufel0406** [comment]
> Me too, best way i choice is linked you EGS to Steam Account and login using steam scan QR Code

it dont work i can log in but the captcha is on the buying screen

**Comment by Teufel0406** [comment]
> are u using windows/linux?

docker on unraid what is linux

**Comment by TheBrokenG0d** [comment]
> > are u using windows/linux?
> 
> docker on unraid what is linux

Unraid is Linux, In the Same Boat with my Docker on Unraid

**Comment by djmulder** [comment]
As explained in the readme on docker it has troubles with the captcha. Best to do the standalone install until it's fixed.

**Comment by djmulder** [comment]
@vogler  

Best to place it here, but it seems even the non-docker version breaks the captcha :/ so that route is also a dead end now

**Comment by deboy69** [comment]
I linked my steam account and changed the browser (in VNC) cookie etc settings from strict to standard. Worked for me.

**Comment by trebllaw** [comment]
yep, running it locally, headless and im presented with the captcha error. Showing the browser also present the same issue.

**Comment by llama0815** [comment]
I tried it solving the captcha through VNC manually and still failed.

**Comment by CoolDouble** [comment]
Did you figure out a fix? I have the same exact problem

**Comment by floxigen** [comment]
Same issue on Ubuntu, I've not the issue on windows

**Comment by ceszkraft** [comment]
Try to change your IP address. It works with docker container

`docker network create --driver bridge free-games-claimer
`
`docker run --rm --network free-games-claimer -p 6080:6080 -v fgc:/fgc/data --pull=always ghcr.io/vogler/free-games-claime`

**Comment by djmulder** [comment]
> Try to change your IP address. It works with docker container
> 
> `docker network create --driver bridge free-games-claimer ` `docker run --rm --network free-games-claimer -p 6080:6080 -v fgc:/fgc/data --pull=always ghcr.io/vogler/free-games-claime`

I still am curious on what ppl mean by this.. Sure I know what IP is.. But I don't really have control over my isp's IP assignment.

Or are people talking about docker v internal v external IP? (judging by you running a docker in bridge)

**Comment by Kaschtelator** [comment]
**German:**
ich weiß nun nicht ob das der richtige fehler Therd ist aber mir ist aufgefallen das bei Epic Games diese meldung immer kommt und ich sie so nicht behoben kriege außer ich claime selbst das spiel.
kann man das irgendwie lösen?

**Englisch:**
I don't know if this is the right error thread, but I noticed that this message always comes up on Epic Games and I can't fix it unless I claim the game myself.
Is there any way to solve this?


`Free games: [
  'https://store.epicgames.com/en-US/p/world-of-warships--starter-pack-albany',
  'https://store.epicgames.com/en-US/p/deaths-gambit-afterlife-b5f411'
]
Current free game: World of Warships — Starter Pack: Albany
  Not in library yet! Click GET.
  Got hcaptcha challenge! Lost trust due to too many login attempts? You can solve the captcha in the browser or get a new IP address.
locator.waitFor: Timeout 60000ms exceeded.
Call log:
  - waiting for locator('text=Thanks for your order!')

    at /fgc/epic-games.js:275:59 {
  name: 'TimeoutError'
}
  Failed to claim! To avoid captchas try to get a new IP address.`

**Comment by vogler** [comment]
Run it without docker for now. Usually I don't get any captchas unless I try around too much. Solving captchas in the browser works for me. Last time I tried it via VNC, solving it also didn't work. Don't know what the issue is there.

**Comment by subcidr** [comment]
Just chippin' in to say that this remains a problem, solving capatcha does not work. Even opening a new tab and trying to log-into Epicgames.com does not work either.

**Comment by IAteYourCookies** [comment]
My Epic Games also sais it has encountered a captcha. My GOG has no such issues.

**Comment by CountOlaf87** [comment]
I'm also having this issue, is there a workaround?

**Comment by vogler** [comment]
See https://github.com/vogler/free-games-claimer/issues/484#issuecomment-3106716661.
The `dev` tag of the Docker image should let you solve the captcha manually.
Workaround is to not use Docker since it creates too many issues. Not getting any captchas with chrome (same as my local browser), however, that's not distributed by playwright for aarch64 Linux.
